### PR TITLE
chore: Dependencies, usings, namespaces, and such

### PR DESCRIPTION
**Pull request by MWO1024**
## Checklist
- [x] Should the change be behind a feature flag?
- [x] Can the feature be meaningfully disabled or circumvented if there are issues (e.g., database-breaking changes)?
- [x] Has it been considered whether data is being delivered to the wrong actor?
- [x] Subsystem test executed (dev_002/dev_003)
  - https://github.com/Energinet-DataHub/dh3-environments/actions/runs/11494720080
- [x] Is there time to monitor state of the release to Production?
- [x] Reference to the task
  - No task, just a little autumn cleaning

### CODE REVIEW

File: source/IntegrationEvents.Application/IntegrationEvents.Application.csproj [hunk 1]
```diff
@@ -14,6 +14,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <PackageReference Include="AspNetCore.HealthChecks.AzureStorage" Version="7.0.0" />
```
Comment (RasmusGraabaek) [inline source/IntegrationEvents.Application/IntegrationEvents.Application.csproj:17]: Indexing

File: source/OutgoingMessages.IntegrationTests/OutgoingMessages.IntegrationTests.csproj [hunk 2]
```diff
@@ -34,11 +34,17 @@
...
         <ProjectReference Include="..\BuildingBlocks.Tests\BuildingBlocks.Tests.csproj"/>
         <ProjectReference Include="..\BuildingBlocks.Infrastructure\BuildingBlocks.Infrastructure.csproj"/>
         <ProjectReference Include="..\Contracts\Contracts.csproj"/>
+        <ProjectReference Include="..\IncomingMessages.Infrastructure\IncomingMessages.Infrastructure.csproj" />
         <ProjectReference Include="..\MasterData.Application\MasterData.Application.csproj" />
+        <ProjectReference Include="..\OutgoingMessages.Application\OutgoingMessages.Application.csproj" />
+        <ProjectReference Include="..\OutgoingMessages.Domain\OutgoingMessages.Domain.csproj" />
+        <ProjectReference Include="..\OutgoingMessages.Infrastructure\OutgoingMessages.Infrastructure.csproj" />
         <ProjectReference Include="..\Process.Domain\Process.Domain.csproj"/>
+        <ProjectReference Include="..\Process.Infrastructure\Process.Infrastructure.csproj" />
```
Comment (RasmusGraabaek) [inline source/OutgoingMessages.IntegrationTests/OutgoingMessages.IntegrationTests.csproj:47]: Why do we need this reference?

**Review by RasmusGraabaek — APPROVED** [review]

File: source/OutgoingMessages.IntegrationTests/OutgoingMessages.IntegrationTests.csproj (hunk 2 above)
Comment (MWO1024) [reply source/OutgoingMessages.IntegrationTests/OutgoingMessages.IntegrationTests.csproj:47]: `.AddTransient<InboxEventsProcessor>()`
and
`.AddScopedSqlDbContext<ProcessContext>(config)`

I'll check what happens if they are removed.

Comment (MWO1024) [reply source/OutgoingMessages.IntegrationTests/OutgoingMessages.IntegrationTests.csproj]: They were not used. I have removed the lines and the ref to Proc.Infra

**Review by RasmusGraabaek — APPROVED** [review]

**Pull request merged**